pipelines/powershell-package/templates/test.yaml

```diff
@@ -0,0 +1,62 @@
+# About this template
+# It contains steps needed for testing powershell module with PSScriptAnalyzer
```

Comment:
I suggest split to 2 templates:
Test (with Pester)
PS Analyze (with PSScriptAnalyzer)